tests/Oryx.Integration.Tests/LocalDockerTests/DatabaseTestsBase.cs

```diff
@@ -19,27 +19,25 @@ public abstract class DatabaseTestsBase
         protected readonly ITestOutputHelper _output;
         protected readonly Fixtures.DbContainerFixtureBase _dbFixture;
         private static readonly Random _rand = new Random();
-        protected readonly int _appPort;
+        protected readonly int _hostPort;
+        protected readonly HttpClient _httpClient = new HttpClient();
 
         protected DatabaseTestsBase(ITestOutputHelper outputHelper, Fixtures.DbContainerFixtureBase dbFixture)
         {
             _output = outputHelper;
             _dbFixture = dbFixture;
-            _appPort = 8080 + _rand.Next(100);
+            _hostPort = 8080 + _rand.Next(100);
```

Comment:
I am wondering how the changes in this PR fixes the issue that we are seeing...I strongly suspect that the issue we might be seeing is due to collisions with the port numbers that are getting generated since uniqueness is not guaranteed here...I think we should just stick to static ports (for now).

Reply:
The errors we've seen were from the DB containers, not the app containers, so I'm compelled to leave this as-is for now. You might be right in general - it's just not something I'd like to tackle in this PR :)

Reply:
Ah I see, got it..